### PR TITLE
Add task to build browser scripts

### PR DESCRIPTION
**Pull request by thammin**
* add `setImmediate` polyfill for browser.
* add `build.js` to build scripts for browser.

btw, maybe we should add a benchmark for browser script since we are using `setImmediate` polyfill as fallback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,28 @@ All Aigle class is extended by same [Aigle-Core](https://github.com/suguru03/aig
...
+Usage
+--
+
+#### node
+```sh
+npm install --save aigle
+```
+
+```js
+var Promise = require('aigle');
```
Comment (suguru03) [inline README.md]: Can you change `var` to `const`?

File: index.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
 'use strict';
 
+require('setimmediate');
```
Comment (suguru03) [inline index.js]: This `index.js` is main root, the library should be in `dependencies`.
But I don't want to add it to `dependencies`, is it possible to use another main root for build?

File: build.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const exec = require('child_process').exec;
```
Comment (suguru03) [inline build.js]: I prefer `const { exec } = require('child_process');` 😄

File: index.js (hunk 2 above)
Comment (thammin) [inline index.js]: You are right, should just include the polyfill when doing bundling.

**Pull request merged**